app/models/concerns/account_finder_concern.rb

```diff
@@ -12,6 +16,10 @@ def find_remote!(username, domain)
       find_remote(username, domain) || raise(ActiveRecord::RecordNotFound)
     end
 
+    def representative
+      find_local(Setting.site_contact_username.gsub(/\A@/, ''))
```

Comment:
I believe the `|| Account.local.find_by(suspended: false)` logic should be included here, since it is repeated everywhere this method is called.